dataprofiler/profilers/profiler_options.py

```diff
@@ -599,8 +599,49 @@ def _validate_helper(self, variable_path='TextOptions'):
             errors.append("{}.vocab must be a BooleanOption."
                           .format(variable_path))
         errors += self.vocab._validate_helper(variable_path + '.vocab')
+
+        if self.properties["num_zeros"].is_enabled:
+            errors.append("{}.num_zeros should always be disabled,"
+                          " num_zeros.is_enabled = False"
+                          .format(variable_path))
+
+        if self.properties["num_negatives"].is_enabled:
+            errors.append("{}.num_negatives should always be disabled,"
+                          " num_negatives.is_enabled = False"
+                          .format(variable_path))
         return errors
 
+    @property
+    def is_numeric_stats_enabled(self):
+        """
+        Returns the state of numeric stats being enabled / disabled. If any
+        numeric stats property is enabled it will return True, otherwise it
+        will return False.
+
+        :return: true if any numeric stats property is enabled, otherwise false
+        :rtype bool:
+        """
+        return super().is_numeric_stats_enabled
```

Comment:
This isn't necessary since this property is already inherited from the parent, you don't need to make a version of it in the child class.

Reply:
In order for the below function, is_numeric_stats_enabled(self, value) to work properly, we need to have a override function is_numeric_stats_enabled(self) inside TextOptions class.

Reply:
Then please add a comment explaining as much since this may seem unnecessary to anyone reading this function.

Reply:
something still needs off here as `super().is_numeric_stats_enabled` itself shouldn't work bc it should not include num_zeros and num_negatives (should be false by default) in which case do we have a test validating that this property works?